Gemfile

```diff
@@ -27,6 +27,9 @@ gem "foreman"
 # Canonical meta tag
 gem "canonical-rails"
 
+# User accounts; required in config/application.rb
+gem 'devise', require: false
+
```

Comment:
When start the real repo, we should take the version numbers from the Gemfile.lock and lock down the Gem versions